content/dei/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Diversity, Equity, and Inclusion"
+date: 2021-09-17
+draft: false
+---
+
+Healthy communities are built when everyone's voice is heard,
+when their perspective is valued, and when their work is recognized.
+Better technical and social solutions can be found through wide participation.
+We are therefore dedicated to expanding our community in a welcoming and inclusive way,
+with particular focus on increased participation from minorities underrepresented in STEM.
```

Comment:
```suggestion
with particular focus on increased participation by people who identify with groups that are  underrepresented in STEM.
```
I'm not sure I like my fix either, but I feel like  calling folk minorities reduces them to one part of their identity.